Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/clean.py

```diff
@@ -41,15 +41,22 @@ def parser(cls, parser, loggers=None):
             type=str, default=None,
             help='String representation(s) of a commit or branch to be cleaned',
         )
+        parser.add_argument(
+            '--remote', dest='remote', type=str, default=None,
+            help='Specify remote to search for pull request from.',
+        )
 
     @classmethod
-    def cleanup(cls, repository, argument):
+    def cleanup(cls, repository, argument, remote_target=None):
         if not repository.is_git:
             sys.stderr('Can only cleanup branches on git repositories\n')
             return 1
 
-        rmt = repository.remote()
-        target = 'fork' if isinstance(rmt, remote.GitHub) else 'origin'
+        rmt = repository.remote(name=remote_target)
```

Comment:
Nit: rmt -> remote (for better readability)

Reply:
That will shadow `from webkitscmpy import remote`